SeleniumTestXUnit/Tests/Features/Item/SetPriority.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Set priority
+    As a logged in user, I want to be able to set priority to my notes
+
+
+    Background:
+        Given the user is logged in
+        And the user has existing notes
+
+    @acceptance
+    Scenario Outline: Set priority to notes
+        Given the user wants to set a priority to the notes
```

Comment:
This step doesn't seem to be needed either.
And just to be sure, by note do you mean item? or is it another type of element that todo.ly lets you create? If it isn't the same, all the notes features should be on a different "Note" folder.